scripts/analyze.sh

```diff
@@ -314,8 +314,10 @@ if check_for_require ocf || check_config_file ZJS_OCF; then
 fi
 
 if check_for_require gpio || check_config_file ZJS_GPIO; then
+    >&2 echo Using module: GPIO
     MODULES+=" -DBUILD_MODULE_GPIO"
     echo "CONFIG_GPIO=y" >> $PRJFILE
+    echo "export ZJS_BOARD=y" >> $CONFFILE
```

Comment:
the check below also set export ZJS_BOARD=y, so you are setting it twice for GPIO?  not that it hurts anything.